src/pages/home/ReportScreen.js

```diff
@@ -328,6 +328,7 @@ class ReportScreen extends React.Component {
                             pendingAction={addWorkspaceRoomOrChatPendingAction}
                             errors={addWorkspaceRoomOrChatErrors}
                             shouldShowErrorMessages={false}
+                            needsOffscreenAlphaCompositing
```

Comment:
Coming from https://github.com/Expensify/App/issues/42757 where this caused an issue for the rendering of the background image on new chats created offline on Android.